apps/sms/services/js/message_manager.js

```diff
@@ -396,26 +500,42 @@ var MessageManager = {
     });
   },
 
+  /**
+   * Method used mark a thread (all messages in it) as read.
+   * @param {number|string} threadId The ID of the thread to be marked as read.
+   * @param {boolean} isRead (optional) Set to true by default, not needed
```

Comment:
i would say, remove 'not needed to be passed' : as it depends on the user requirement. want to mark as read-> no need to pass, if want to mark as unread -> need to pass 'false'